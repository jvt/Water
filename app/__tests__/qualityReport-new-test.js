import {
	Image,
	Text,
	TextInput,
} from 'react-native';
import React from 'react';
import QualityReportNew from '../qualityReport/new';
import Button from 'apsl-react-native-button';
import Renderer from 'react-test-renderer';

describe('AdminPanel', () => {

	test('DropletImage', () => {
		const instance  = Renderer.create(
			<Image 
			style={{}}
			Droplet
			/>

		);

		expect(instance.toJSON()).toMatchSnapshot();
	});

	test('TitleText', () => {
		const instance  = Renderer.create(
			<Text 
			style={{}}>
			New Water Report
			</Text>

		);

		expect(instance.toJSON()).toMatchSnapshot();
	});

	test('VirusPPMInput', () => {
		const instance  = Renderer.create(
			<TextInput
			style={{}}
			VirusPPM
			/>

		);

		expect(instance.toJSON()).toMatchSnapshot();
	});

	test('ContaminantPPMInput', () => {
		const instance  = Renderer.create(
			<TextInput
			style={{}}
			ContaminantPPM
			/>

		);

		expect(instance.toJSON()).toMatchSnapshot();
	});

	test('SubmitButton', () => {
		const instance  = Renderer.create(
			<Button 
			style={{}}>
			Submit
			</Button>

		);

		expect(instance.toJSON()).toMatchSnapshot();
	});

	test('CancelButton', () => {
		const instance  = Renderer.create(
			<Button 
			style={{}}>
			Cancel
			</Button>

		);

		expect(instance.toJSON()).toMatchSnapshot();
	});

});
