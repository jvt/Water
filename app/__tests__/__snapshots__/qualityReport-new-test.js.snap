exports[`AdminPanel CancelButton 1`] = `
<View
  accessibilityComponentType={undefined}
  accessibilityLabel={undefined}
  accessibilityTraits={undefined}
  accessible={true}
  hitSlop={undefined}
  isTVSelectable={true}
  onLayout={undefined}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "alignSelf": "stretch",
      "borderRadius": 8,
      "borderWidth": 1,
      "flexDirection": "row",
      "height": 44,
      "justifyContent": "center",
      "marginBottom": 10,
      "opacity": 1,
    }
  }
  testID={undefined}
  tvParallaxProperties={undefined}>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "backgroundColor": "transparent",
          "flex": 1,
          "fontSize": 18,
          "textAlign": "center",
        },
        undefined,
      ]
    }>
    Cancel
  </Text>
</View>
`;

exports[`AdminPanel ContaminantPPMInput 1`] = `
<TextInput
  ContaminantPPM={true}
  style={Object {}} />
`;

exports[`AdminPanel DropletImage 1`] = `
<Image
  Droplet={true}
  style={Object {}} />
`;

exports[`AdminPanel SubmitButton 1`] = `
<View
  accessibilityComponentType={undefined}
  accessibilityLabel={undefined}
  accessibilityTraits={undefined}
  accessible={true}
  hitSlop={undefined}
  isTVSelectable={true}
  onLayout={undefined}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "alignSelf": "stretch",
      "borderRadius": 8,
      "borderWidth": 1,
      "flexDirection": "row",
      "height": 44,
      "justifyContent": "center",
      "marginBottom": 10,
      "opacity": 1,
    }
  }
  testID={undefined}
  tvParallaxProperties={undefined}>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "backgroundColor": "transparent",
          "flex": 1,
          "fontSize": 18,
          "textAlign": "center",
        },
        undefined,
      ]
    }>
    Submit
  </Text>
</View>
`;

exports[`AdminPanel TitleText 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
  style={Object {}}>
  New Water Report
</Text>
`;

exports[`AdminPanel VirusPPMInput 1`] = `
<TextInput
  VirusPPM={true}
  style={Object {}} />
`;
